examples/tasks_tutorial.py

```diff
@@ -142,3 +142,69 @@
 ids = [2, 1891, 31, 9983]
 tasks = openml.tasks.get_tasks(ids)
 pprint(tasks[0])
+
+############################################################################
+# Creating tasks
+# ^^^^^^^^^^^^^^^^^
+#
+# You can also create new tasks. Take the following into account:
+#
+# * You can only create tasks on _active_ datasets
+# * For now, only the following tasks are supported: classification, regression,<br>
```

Comment:
Is the `<br>` necessary to produce good layout? I'd prefer to not have html tags in there as the bootstrap layout of the website or the jupyter layout should take care of breaking the lines, right?